.github/stale.yml

```diff
@@ -1,11 +1,12 @@
 # Number of days of inactivity before an issue becomes stale
-daysUntilStale: 90
+daysUntilStale: 120
 # Number of days of inactivity before a stale issue is closed
-daysUntilClose: 20
+daysUntilClose: 30
 # Issues with these labels will never be considered stale
 exemptLabels:
   - pinned
   - security
+  - todo
```

Comment:
the effect of this is that I'll just prompt you to add `todo` to any issue I open

Reply:
that's not an obvious problem to me.

Reply:
it seems backwards that high quality reports should carry the burden of requesting the "good issue" label, arguing why they deserve it, knowing to do so in the first place, re-requesting it if you forget to apply the label when responding to the ticket, and dealing with the additional noise.

I make high quality reports and I want all the burden to be on low-quality reporters :)